src/Components/SideNav.jsx

```diff
@@ -1,15 +1,48 @@
-import { Box, Typography } from "@mui/material";
+import { Box, List, ListItem, ListItemButton, Typography } from "@mui/material";
+import { useNavigate, useLocation } from "react-router-dom";
+import { routes } from "./Utils/routes";
 
 export const SideNav = () => {
+  const navigate = useNavigate();
+  const location = useLocation();
+
   return (
-    <Box sx={{ display: { xs: "none", md: "flex" } }}>
-      <Typography
-        variant="h2"
-        color="initial"
-        sx={{ background: "red", height: "calc(100vh - 64px)" }}
+    <Box
+      sx={{
+        width: "200px",
+        display: { xs: "none", md: "flex" },
+      }}
+      role="presentation"
+      borderRight={1}
+      borderColor="#f3e5f5"
+    >
+      <List
```

Comment:
List component seems to be repeating. Extracting it as a seperate component will keep code clean

Reply:
Yep. Thankyou bro. Extracted it as a component.